cocotb_build_libs.py

```diff
@@ -268,7 +268,7 @@ def _get_common_lib_ext(include_dir, share_lib_dir):
         extra_link_args=_extra_link_args(rpath="$ORIGIN/libs"),
     )
 
-    return [libcocotbutils, libgpilog, libcocotb, libgpi, libsim]
```

Comment:
If this order matters, a comment saying why it matters would be nice.

Reply:
cocotbutils depends on gpilog, so gpilog must be built first. Libraries are built in order.